src/filehandler/pemhandler.cpp

```diff
@@ -1,44 +1,43 @@
-#include "pemhandler.h"
-#include "utils/base64.h"
-#include "utils/hex.h"
+#include "filehandler/pemhandler.h"
+#include "base64.h"
+#include "hex.h"
 
 #include <fstream>
 #include <iostream>
 #include <sodium.h>
 
 namespace ih
 {
-PemInputHandler::PemInputHandler(wrapper::PemHandlerInputWrapper const inputWrapper) :
-        IFileHandler(inputWrapper.getPemFilePath()),
-        m_inputData(inputWrapper)
+PemFileReader::PemFileReader(std::string const filePath) :
+        IFileHandler(filePath)
 {
     m_fileContent = getFileContent();
 }
 
-bool PemInputHandler::isFileValid() const
+bool PemFileReader::isFileValid() const
 {
     return IFileHandler::fileExists() &&
-           IFileHandler::isFileExtensionValid("pem") &&
+            IFileHandler::isFileExtension("pem") &&
            (m_fileContent != "");
 }
 
-Address PemInputHandler::getAddress() const
+Address PemFileReader::getAddress() const
 {
     bytes keyBytes = getKeyBytesFromFile();
-    keyBytes.erase(keyBytes.begin(), keyBytes.begin() + 32);
+    keyBytes.erase(keyBytes.begin(), keyBytes.begin() + crypto_sign_PUBLICKEYBYTES);
```

Comment:
_... NUM ..._ or _... LENGTH ..._ - perhaps redefine a constant with a more explicit name, perhaps in a file where we can keep all our constants?

Reply:
Good point. I also don't like using those defined values from external library `libsodium`.
In [this](https://elrondnetwork.atlassian.net/browse/EN-10130) ticket I took care of them.